Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -583,6 +596,94 @@ private static void Clean(NugetPackageIdentifier package)
             }
         }
 
+        private static string GetNuGetVersionName(string directoryName)
+        {
+            string versionName = "";
+
+            if (directoryName == "netstandard2.0")
```

Comment:
I'm hoping we can avoid this whole mapping (including the need to maintain it), rephrase the log slightly to leverage the directory names themselves and point users here: https://docs.microsoft.com/en-us/nuget/reference/target-frameworks?

Reply:
And potentially here https://docs.unity3d.com/ScriptReference/ApiCompatibilityLevel.html

Reply:
OK. I got your mean. I fixed the redundancy judgment.

Reply:
Can you delete this method now that no one calls it?

Reply:
Can you remove this function now that it's unused?